src/test/java/org/opentripplanner/middleware/ApiUserFlowTest.java

```diff
@@ -191,4 +194,48 @@ public void canSimulateApiUserFlow() {
         ApiUser deletedApiUser = Persistence.apiUsers.getById(apiUser.id);
         assertNull(deletedApiUser);
     }
+
+    /**
+     * Tests for the chain of API calls to verify a user's phone number.
+     */
+    @Test
+    public void canSimulatePhoneVerificationFlow() {
```

Comment:
I don't think this test really belongs here. This test class was intended to model how a third party API user interacts with otp-middleware. And an API user (eventually) won't be able to create an OtpUser that can register a phone number for SMS notifications.

Reply:
I moved and changed the test in https://github.com/ibi-group/otp-middleware/pull/83/commits/44864ac4f129dd0e1bc766444a85c73776583201.